user_guide_src/source/outgoing/view_parser/014.php

```diff
@@ -1,8 +1,12 @@
 <?php
 
-public $plugins = [
-    'foo' => '\Some\Class::methodName',
-    'bar' => function ($str, array $params=[]) {
-        return $str;
-    },
-];
+class View extends BaseView
+{
+
+    public $plugins = [
+        'foo' => '\Some\Class::methodName',
+        'bar' => function($str, array $params = []) {
+            return $str;
+        },
```

Comment:
Apparently, this does not work, but need to use the constructor instead:
https://www.haykranen.nl/2013/03/13/howto-use-closure-functions-as-array-values-in-a-class-in-php-5-4/

Reply:
Actually, the docs themselves say the exact same thing in example 015 two lines later.
So I decided to remove the closure here.